data/geonode_regions_to_administrative_divisions_code.csv

```diff
@@ -0,0 +1,4 @@
+"Turkey",233,249
+"Indonesia",115,116
+"Global",1,116
+"Global",1,249
```

Comment:
As discussed with @fvanderbiest, it would be better to manage Global further resources in a different way. The "Global" region applies to all the administrative divisions. We then probably don't need to have it in the current lookup table.